kobo/apps/stripe/tests/test_one_time_addons_tracking.py

```diff
@@ -0,0 +1,196 @@
+from dateutil.relativedelta import relativedelta
+from django.contrib.auth.models import User
+from django.core.cache import cache
+from django.urls import reverse
+from django.utils import timezone
+from djstripe.models import (
+    Charge,
+    Customer,
+    PaymentIntent,
+    Price,
+    Product,
+    Subscription,
+    SubscriptionItem,
+)
+from model_bakery import baker
+
+from kobo.apps.organizations.models import Organization, OrganizationUser
+from kobo.apps.trackers.models import NLPUsageCounter
+from kobo.apps.trackers.submission_utils import (
+    create_mock_assets,
+)
+from kobo.apps.stripe.models import PlanAddOn
+from kobo.apps.trackers.utils import update_nlp_counter
+from kpi.tests.api.v2.test_api_service_usage import ServiceUsageAPIBase
+
+
+class OneTimeAddonTrackingTestCase(ServiceUsageAPIBase):
```

Comment:
This is still an interim test class at the moment, but I do wonder how I should clean it up once it is ready. There is a fair amount of overlap with `ServiceUsageAPIBase` and `OrganizationUsageAPITestCase`. Ideally, I think I would create a base test class with methods flexible enough to work in all three classes. But that might be better left to another PR.